src/components/rocp_sdk/tests/Makefile

```
@@ -1,6 +1,7 @@
 NAME=template
 include ../../Makefile_comp_tests.target
 
+AMDCXX    = amdclang++
```

To allow the user to set this variable to override the default, I think you need:
```suggestion
AMDCXX    := amdclang++
```

---

You are right. I fixed it.